src/database/connection.rs

```diff
@@ -16,14 +16,32 @@ pub enum DatabaseConnection {
 
 pub type DbConn = DatabaseConnection;
 
+pub trait IntoDbBackend {
```

Comment:
I think there is actually no need to introduce a new trait here.
There is only 1 impl and it's not being used in any trait bound.
It might have been useful before, but seemingly not in this final solution.
So I'd suggest moving the methods back to its original `impl DbBackend` block.

Reply:
Removed `IntoDbBackend `,roll back to original impl DbBackend block.